### ISSUE TITLE
Prevent entering integers / numbers for city & region (text input @ theme)

### ISSUE THREAD

**Issue by dev-101** [osclass/Osclass#2071]
original forum topic here:

http://forums.osclass.org/3-6-x/mozilla5-0-(windows-u-windows-nt-5-1-en-us-rv1-9-0-12)-gecko2009070611-fi/

1. disable js
2. use text inputs for locations(bender)
3. post new item, enter some integers for city & region instead of letters

This should not happen.

**Comment by garciademarina** [comment]
First, Osclass needs javascript in order to work properly.

Second, locations of any type, country, region, city, city_area, can be alphanumeric so this should not be an issue.

**Comment by dev-101** [comment]
Yeah, though of it later and updated issue description above, so it was done on purpose.

Anyway, this is personally not an issue for me, but the user who posted it obviously do not wish to allow numbers for city and region (also, bad bots usually do not run javascript, or execute it selectively in their favor to bypass validation - and I highly suspect this was posted by a bot).

**Comment by garciademarina** [comment]
If we can reproduce the issue using Bender theme we will try to fix it.

**Comment by dev-101** [comment]
I reproduced it already, it is indeed real.
But, for compatibility, this needs to be added as an admin option.

**Comment by garciademarina** [comment]
Locations of any type, country, region, city, city_area, can be alphanumeric so this should not be an issue. If you are using autocomplete locations user can add numbers as a location. 

If you are using dropdown locations you shouldn't have this issue.

**Comment by dev-101** [comment]
Yes, with drop-down internally numbers are tested and replaced by null in that case, only with auto-complete this could be a problem. If user has only one or few countries where regions are only with letters, this could improve internal security of the script by not allowing numbers.

However, I see that serjuc11111 has this problem with drop-downs (which confuses me, as I cannot reproduce that).

**Comment by garciademarina** [comment]
May be theme related

**Comment by dev-101** [comment]
Possibility, regardless, this could be implemented in the admin as an optional checkboxes (allow letters, allow numbers). Think about it...
